### ISSUE TITLE
7장 시계열 데이터

### ISSUE THREAD

**Issue by blue7u** [PinkWink/DataScience#14]
![error](https://user-images.githubusercontent.com/40557595/41833316-cadabb5c-788a-11e8-97ac-7a794efeaaac.PNG)

위와 같은에러가 뜹니다.

pip install도 책에 있는것과 동일하게 했구요..

뭐가 문제일까요..?

**Comment by PinkWink** [comment]
jupyter notebook 상에서 
!conda list
를 실행하고, 그 결과에 설치한 모듈이 있는지 보세요...
간혹 여러개의 Python을 사용하시는 경우 문제가 있을 수 있더라구요...

**Comment by pepitto** [comment]
안녕하세요, fpprophet을 이용한 시계열 데이터 분석이 상당히 유용해 보입니다.
그런데 실제 예측결과의 정확성을 수치적으로 판단하려면 어떤 방법이 좋을지요?

**Comment by PinkWink** [comment]
예측이니... 실제로 잘 맞췄는지^^ 비교하면 좋겠죠^^

만약 예를들어 2년6개월치 데이터를 확보했다면
2년치를 가지고, 그 다음 1개월
2년 1개월치를 가지고, 또 그다음 1개월...
을 예측해 본다면... 참값과 비교를 할 수 있을테니

에러의 MSE를 계산해 둘 수 있지 않을까요???

**Comment by pepitto** [comment]
네 저도 그렇게 해보려고 하는데요...교재에서는 2014년부터 2016.5.31까지 데이터를 61일간 예측하셨는데요.
그럼 2016.6~7월까지의 데이터를 예측해서 실제값과 비교하면 되는건지요?
교재의 real/forecast 비교그래프 상으로는 5월말까지만 있는거 같아서 좀 햇갈리네요.

**Comment by lolhjlolhj** [comment]
fbprophet 설치관련하여 질문이 있습니다.

터미널 : pip install pystan 과 pip install prophet (설치완료)
아나콘다 프롬프트 창 : conda install -c conda-forge fbprophet (설치완료)

하지만 다음과 같이 에러가 발생합니다. 
(ValueError: More than one filename match found for utility code Capsule
--> 이런 에러 메세지는 구글에 검색해도 안나오는 것 같더군요..)

![error](https://user-images.githubusercontent.com/40727185/48348113-b2592d00-e6c3-11e8-84af-d4c0989fa95d.png)

그래서 찬찬히 다시 책을 읽어보니 Visual C ++ Build Tools를 설치를 안했더군요.

말씀해주신 https://goo.gl/22QbmB를 접속해봤는데 말씀해주신 곳의 링크가 깨져서 나옵니다.

어디서 Visual C ++ Build Tools를 설치하면 되는지요...
